website/documentation/050-tutorials/content/howto/shoot_istio_dns_certs/_index.md

```diff
@@ -0,0 +1,464 @@
+* [CRUD Gardener Shoot](https://gardener.cloud/documentation/050-tutorials/content/howto/create-delete-shoot/)
+* [DNS Management](https://gardener.cloud/documentation/050-tutorials/content/howto/gardener_dns_management/)
+* [Certificate Mangement](https://gardener.cloud/documentation/050-tutorials/content/howto/gardener_certificate_management/)
+* [Tutorial Domain Names](https://gardener.cloud/documentation/050-tutorials/content/howto/dns_names/)
+* [Tutorial Certifictates](https://gardener.cloud/documentation/050-tutorials/content/howto/x509_certificates/)
+
+{{% notice tip %}}
+<p>If you try my instructions and fail, then read the alternative title of this tutorial as "Shoot yourself in the foot with Gardener, custom Domains, Istio and Certificates".</p>
+{{% /notice %}}
+
+## First Things First
+
+Login to your Gardener landscape, setup a project with adequate infrastructure credentials and then navigate to your account. Note down the name of your secret. I chose the GCP infrastructure from the vast possible options that my Gardener provides me with, so i had named the secret as `shoot-operator-gcp`. 
+
+From the Access widget (leave the default settings) download your personalized `kubeconfig` into `~/.kube/kubeconfig-garden-myproject`. Follow the instructions to setup `kubelogin`:
+![access](./access.png)
+For convinience, let us set an alias command with 
+```bash
+alias kgarden="kubectl --kubeconfig ~/.kube/kubeconfig-garden-myproject.yaml"
+``` 
+`kgarden` now gives you all botanical powers and connects you directly with your Gardener.
+
+You should now be able to run `kgarden get shoots`, automatically get an oidc token, and list already running clusters/shoots.
+
+## Prepare your Custom Domain
+I am going to use [Cloud Flare](https://www.cloudflare.com/) as programmatic DNS of my custom domain `mydomain.io`. Please follow detailed instructions from Cloud Flare on how to delegate your domain (the free account does not support delegating subdomains). Alternatively, AWS Route53 (and most others) support [delegating subdomains](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html).
+
+I needed to follow these [instructions](https://github.com/gardener/external-dns-management/blob/master/doc/cloudflare/README.md) and created the following secret:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloudflare-mydomain-io
+type: Opaque
+data:
+  CLOUDFLARE_API_TOKEN: useYOURownDAMITzNDU2Nzg5MDEyMzQ1Njc4OQ==
+```
+Apply this secret into your project with `kgarden create -f cloudflare-mydomain-io.yaml`.
+
+Our [External DNS Manager](https://github.com/gardener/external-dns-management/) also supports Amazon Route53, Google CloudDNS, AliCloud DNS, Azure DNS, or OpenStack Designate. Check it out.
+
+## Prepare Gardener Extensions
+I now need to prepare the Gardener extensions `shoot-dns-service` and `shoot-cert-service` and set the parameters accordingly.
+
```

Comment:
{{% notice note %}}
<p>Please note, that the availability of Gardener Extensions depend on the Gardener installation you use. Please contact your Gardener administrator in case you experience any issues during activation.</p>
{{% /notice %}}